src/core_plugins/kibana/public/management/sections/indices/_create.js

```diff
@@ -247,6 +254,19 @@ uiModules.get('apps/management')
     }, notify.fatal);
   }
 
+  function updateFieldListAndSeed(results, seedField) {
+
+    updateFieldList(results);
+
+    if (!results.dateFields.length) {
+      return;
+    }
+
+    const matchingField = results.dateFields.find(field => field.name === seedField.name);
+    index.timeField = matchingField ? matchingField : results.dateFields[0];
+
+  }
```

Comment:
1 minor nit: I haven't generally seen function bodies with whitespace padding in the codebase, and I know the AirBnB style-guide complains about it by default. It's up to you, but it might be a little more consistent/future-proof to remove the padding.